### PR TITLE
update wgpu

### PR DESCRIPTION
**Pull request by msparkles**

### CODE REVIEW

File: crates/bootstrap/Cargo.toml [hunk 1]
```diff
@@ -19,7 +19,6 @@ image = { version = "0.24.2", default-features = false, features = ["png"] }
 log = "0.4.17"
 pollster = "0.3.0"
 profiling = "1.0.6"
-tracy-client = { version = "0.15.1", optional = true }
-
+tracy-client = { version = "0.17.0", optional = true }
+wgpu = "22.0.0"
```
Comment (LPGhatguy) [inline crates/bootstrap/Cargo.toml:23]: Why also upgrade tracy-client here? Also, can we keep the alphabetical sorting of dependencies?

Comment (msparkles) [reply crates/bootstrap/Cargo.toml:23]: Will fix the ordering, and we accidentally included the updated tracy-client from our fork it seems, oops!

**Comment by LPGhatguy** [comment]
Any idea what's going on with the rustfmt job? I'm surprised that dependencies are failing there. O_o

**Comment by msparkles** [comment]
It's a wgpu issue, a commit is pushed to upstream but not released yet

**Comment by msparkles** [comment]
recommited the thing to trigger CI, lol

**Review by LPGhatguy — APPROVED** [review]
LGTM!

**Pull request merged**